src/test/java/net/openhft/chronicle/queue/impl/single/stress/backwardstailer/BackwardsTailerBoundaryTest.java

```diff
@@ -0,0 +1,105 @@
+package net.openhft.chronicle.queue.impl.single.stress.backwardstailer;
+
+import net.openhft.chronicle.core.io.IOTools;
+import net.openhft.chronicle.core.time.SetTimeProvider;
+import net.openhft.chronicle.queue.*;
+import net.openhft.chronicle.queue.impl.single.SingleChronicleQueue;
+import net.openhft.chronicle.queue.impl.single.SingleChronicleQueueBuilder;
+import org.jetbrains.annotations.NotNull;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(Parameterized.class)
+public class BackwardsTailerBoundaryTest extends ChronicleQueueTestBase {
+
+    private static final Logger log = LoggerFactory.getLogger(BackwardsTailerBoundaryTest.class);
+
+    private SetTimeProvider timeProvider;
+
+    private final RollCycle rollCycle;
+
+    public BackwardsTailerBoundaryTest(RollCycle rollCycle) {
+        this.rollCycle = rollCycle;
+    }
+
+    @Parameterized.Parameters(name = "{0}")
+    public static Collection<Object[]> data() {
+        final List<Object[]> data = new ArrayList<>();
+        data.add(new Object[]{RollCycles.TEST4_DAILY});
```

Comment:
Not so much parameters :) 
I'd recommend to use a JUnit5 parameterized test (slowly migrating of Junit4)

Reply:
:-) Junit5 would have been nicer!